packages/components/src/Clickable/Clickable.tsx

```diff
@@ -37,12 +37,14 @@ function Clickable<IComponent extends React.ElementType>({
   size,
   state,
   variant,
+  className,
   ...rest
 }: ClickableProps<IComponent>) {
   const Component = as;
   return (
     <Component
       className={clsx(
+        className,
```

Comment:
This is where the `className` was breaking. The prop is already being passed down properly using `...rest`, but it was being applied to `Clickable` *instead of* the internal styles.